src/processes/helper/createEmptyChannel.js

```diff
@@ -0,0 +1,29 @@
+const {CHANNEL_TYPE_STRING} = require('../../utils');
+const {LogError} = require('../../databases/models');
+const StreamChat = require('stream-chat').StreamChat;
+
+const createEmptyChannel = async (userId, usersFollowed) => {
+  try {
+    const client = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+
+    await Promise.all(
+      usersFollowed.map(async (userFollowed) => {
+        const members = [userId, userFollowed];
+        const channel = await client.channel(CHANNEL_TYPE_STRING.CHAT, {
+          members,
+          created_by_id: userId
+        });
+
+        const createdChannel = await channel.create();
+        console.info('Created Channel:', createdChannel.channel.id);
+      })
+    );
+  } catch (error) {
+    await LogError.create({
+      message: error.message
+    });
+    console.error('createEmptyChannel: ', error);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling is done correctly here, but it would be better to throw the error after logging it so that the caller of this function can decide how to handle it.

```diff
-    console.error('createEmptyChannel: ', error);
+    throw error;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->